content/docs/user-guide/assets/builder/process-job.md

```diff
@@ -1,167 +1,132 @@
-```
-
-## ProductDependency 
-
-The product dependency information that the builder sends to **Asset Processor** to indicate that a product asset depends on another product asset during load.
+### ProductDependency 
 
-**azlmbr.asset.builder.ProductDependency**
+`azlmbr.asset.builder.ProductDependency` contains product dependency information that the Asset Builder sends to Asset Processor to indicate that a product asset depends on another product asset when loaded or packaged.
 
-```
-class azlmbr.asset.builder.ProductDependency
-- dependencyId (azlmbr.math.Uuid) Asset ID of this product asset dependency
-```
+| Field | Type | Description |
+| - | - | - |
+| `dependencyId` | `azlmbr.math.Uuid` | The asset ID of this product asset dependency. |
 
-## ProductPathDependency 
+### ProductPathDependency 
 
-The `ProductPathDpendency` represents the product's dependency information that the builder detected on another product file (relative to the source asset path). If the source asset ID can be determined, we recommend that you use the `productDependencies` instead to indicate the product's dependency information in terms of asset IDs. It's preferable to depend on product files whenever possible, to avoid introducing unintended dependencies.
+`azlmbr.asset.builder.ProductPathDependency` represents the product's dependency information that the Asset Builder detected on another product asset (relative to the source asset path). If the source asset ID can be determined, we recommend that you use the `productDependencies` instead to indicate the product's dependency information in terms of asset IDs. It's preferable to depend on product assets whenever possible, to avoid introducing unintended dependencies.
 
-The `dependencyType` field indicates if the path points to a source file or a product file.
+The `dependencyType` field indicates if the path points to a source asset or a product asset.
 
-**azlmbr.asset.builder.ProductPathDependency**
+| Field | Type | Description |
+| - | - | - |
+| `dependencyPath` | String | relative path to the asset dependency |
+| `dependencyType` | `azlmbr.asset.builder.ProductPathDependency Type` | Indicates if the dependency path points to a source asset or a product asset. |
 
-```
-class azlmbr.asset.builder.ProductPathDependency
-- dependencyPath (str) Relative path to the asset dependency
-- dependencyType (azlmbr.asset.builder.ProductPathDependency Type) Indicates if the dependency path points to a source file or a product file
-```
+### ProductPathDependency Type 
 
-## ProductPathDependency Type 
+`azlmbr.asset.builder.ProductPathDependency Type` indicates how to use the dependency path in the `ProductPathDependency`. A dependency on a source asset is converted into dependencies on all product assets produced. It's preferred to depend on product assets whenever possible, to avoid introducing unintended dependencies.
```

Comment:
```suggestion
`azlmbr.asset.builder.ProductPathDependency Type` indicates how to use the dependency path in the `ProductPathDependency`. A dependency on a source asset is converted into dependencies on all product assets produced. It's preferable to depend on product assets whenever possible, to avoid introducing unintended dependencies.
```